dartagnan/src/test/java/com/dat3m/dartagnan/spirv/SpirvTest.java

```diff
@@ -33,62 +25,50 @@
-import static com.dat3m.dartagnan.utils.Result.FAIL;
-import static com.dat3m.dartagnan.utils.Result.PASS;
 import static org.junit.Assert.assertEquals;
 
 @RunWith(Parameterized.class)
 public class SpirvTest {
 
+    private static Map<String, Result> expectedResults;
     // TODO: Replace with Vulkan when RA is fixed
     private final String modelPath = getRootPath("cat/sc.cat");
     private final String programPath;
     private final Result expected;
-    private final String grid;
-    private final String specType;
-    private final String spec;
 
-    public SpirvTest(String file, Result expected, String grid, String specType, String spec) {
-        this.programPath = getTestResourcePath("spirv/" + file);
+    public SpirvTest(String file, Result expected) {
+        this.programPath = file;
         this.expected = expected;
-        this.grid = grid;
-        this.specType = specType;
-        this.spec = spec;
     }
 
-    @Parameterized.Parameters(name = "{index}: {0}, file={1}, expected={2}, grid={3}, specType={4}, spec={5}")
+    @Parameterized.Parameters(name = "{index}: {0}, {1}")
     public static Iterable<Object[]> data() throws IOException {
-        return Arrays.asList(new Object[][]{
-                {"empty.spv.dis", PASS, "1,1,1", ASSERT_TYPE_FORALL, "true"},
-                {"empty.spv.dis", PASS, "1,1,1", ASSERT_TYPE_EXISTS, "true"},
-                {"empty.spv.dis", FAIL, "1,1,1", ASSERT_TYPE_NOT_EXISTS, "true"},
-                {"empty.spv.dis", FAIL, "1,1,1", ASSERT_TYPE_FORALL, "false"},
-                {"empty.spv.dis", FAIL, "1,1,1", ASSERT_TYPE_EXISTS, "false"},
-                {"empty.spv.dis", PASS, "1,1,1", ASSERT_TYPE_NOT_EXISTS, "false"},
-                {"init.spv.dis", PASS, "1,1,1", ASSERT_TYPE_FORALL, "%v1=7 %v2=123 %v3=0"},
-                {"read-write.spv.dis", PASS, "1,1,1", ASSERT_TYPE_FORALL, "%v1=2 %v2=1"},
-                {"vector-init.spv.dis", PASS, "1,1,1", ASSERT_TYPE_FORALL, "%v3v[0]=2 %v3v[8]=1 %v3v[16]=0"},
-                {"vector.spv.dis", PASS, "1,1,1", ASSERT_TYPE_FORALL, "%v3v[0]=0 %v3v[8]=1 %v3v[16]=2"},
-                {"vector-read-write.spv.dis", PASS, "1,1,1", ASSERT_TYPE_FORALL, "%v3v[0]=2 %v3v[8]=1 %v3v[16]=0"},
-                {"ids.spv.dis", PASS, "2,2,2", ASSERT_TYPE_FORALL,
-                        "%out[0]=0 %out[8]=0 %out[16]=0 " +
-                                "%out[24]=1 %out[32]=0 %out[40]=0 " +
-                                "%out[48]=0 %out[56]=1 %out[64]=0 " +
-                                "%out[72]=1 %out[80]=1 %out[88]=0 " +
-                                "%out[96]=0 %out[104]=0 %out[112]=1 " +
-                                "%out[120]=1 %out[128]=0 %out[136]=1 " +
-                                "%out[144]=0 %out[152]=1 %out[160]=1 " +
-                                "%out[168]=1 %out[176]=1 %out[184]=1"},
-        });
+        String testPath = "dartagnan/src/test/resources/spirv/";
```

Comment:
Would you mind keeping the list of tests explicit here? I would like to see what it runs for handmade tests.

return Arrays.asList(new Object[][]{
                {"empty-exists.spv.dis", PASS},
                {"empty-exists-false.spv.dis", PASS},
                {"empty-forall.spv.dis", FAIL},
                {"empty-forall-false.spv.dis", FAIL},
                {"empty-not-exists.spv.dis", FAIL},
                {"empty-not-exists-false.spv.dis", PASS},
...

Reply:
Sure, I commeneted out the reading expectated.csv part, is it okay?

Reply:
I would drop reading expected values here completely. We can use it for a set of litmus tests in spirv under test/java/com/dat3m/dartagnan/litmus. Once we have them.

Reply:
I agree with Natalia. For litmus tests we were using the CVS file because originally  (all wmm except the gpu ones) those results were automatically generated by the herd7 tool. For the litmus tests for GPU models we followed this for consistency. At least for now, the expected results of all these tests need some manual inspection because we don't have any other tool to double check the results